src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -5577,7 +5589,17 @@ switch_status_t config_sofia(sofia_config_t reload, char *profile_name)
 						if (v >= 0) {
 							profile->rtp_timeout_sec = v;
 							switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_WARNING,
-											  "rtp-timeout-sec deprecated use media_timeout variable.\n"); 
+											  "rtp-timeout-sec deprecated use rtp-media_timeout variable.\n");
+						}
+					} else if (!strcasecmp(var, "rtp-media-timeout-sec") && !zstr(val)) {
+						int v = atoi(val);
+						if (v >= 0) {
+							profile->rtp_timeout_sec = v;
```

Comment:
Create new variable for rtp media timeout. Take note that rtp-media-timeout is in millisecond format